tests/functional/cocoapods/test_resolvers_cocoapods.py

```diff
@@ -124,14 +123,18 @@ def _version_in_specset(version, specset):
 
 
 def _safe_json_load(filename):
-    # Some fixtures has comments so the stdlib implementation doesn't work.
-    # We only use commentjson if we absolutely need to because it's SLOW.
-    try:
-        with open(filename) as f:
+    # Some fixtures have comments so they are not valid json.
+    # We could use commentjson/json5 to load them,
+    # but it's easier to strip the comments.
+    # We only do it when json.load() fails to avoid unnecessary loading
+    # all the json files to strings.
```

Comment:
nit: This can be communicated more concisely.

```suggestion
    # Some fixtures have comments, so strip them if first parse fails.
    # This is done to avoid loading all JSON files to string before parsing.
```

Reply:
I don't consider the second line intelligible in this context. What does *This* refer to?

What about:

```suggestion
    # Some fixtures have comments, so strip them if first parse fails.
    # We only do this in case of failure to avoid loading all JSON files to strings before parsing.
```